compiler/noirc_frontend/src/ast/statement.rs

```diff
@@ -330,18 +335,11 @@ impl Path {
     }
 
     pub fn from_ident(name: Ident) -> Path {
-        Path { segments: vec![name], kind: PathKind::Plain }
+        Path { span: name.span(), segments: vec![name], kind: PathKind::Plain }
```

Comment:
Why do paths need explicit spans now?

Reply:
Perhaps, we could simply take a different approach to formatting the Path